src/main/steps/common/documents/view/all-documents/template.njk

```diff
@@ -46,4 +46,4 @@
       </li>
     {% endfor %}
   </ol>
-{% endblock %}
\ No newline at end of file
+{% endblock %}
```

Comment:
why there are changes in src/main/steps/common/documents/view/all-documents/template.njk file?

Reply:
removed extra spacing shouldn't be an issue